.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
Reserving this comment to block merge until ci has been manually tested, see @RonTuretzky 's comment above

Reply:
@bennypowers Flake8 detected a line that was too long 

![image](https://user-images.githubusercontent.com/74178515/202418679-038c2c79-abcb-4721-957e-1f162ad89257.png)

proceeding to next steps